src/dosbox.cpp

```diff
@@ -1239,6 +1239,13 @@ void DOSBOX_Init()
 	        "tab-separated format, used by SETVER.EXE as a persistent storage\n"
 	        "(empty by default).");
 
+	pbool = secprop->Add_bool("file_locking", only_at_start, true);
+	pbool->Set_help(
+	        "Enable file locking (SHARE.EXE emulation).\n"
```

Comment:
Put "enabled by default" in the first line:

```
Enable file locking (SHARE.EXE emulation; enabled by default).
```

Reply:
@weirddan455, just a question - does `only_at_start` mean you have one-shot to set it on startup, and then can't toggle it futher during runtime? Is it possible to allow that? (would be cool the let this be switched at runtime for those folks to pop in and out of Windows 3.1, etc).

I guess in the big picture, `enabled` is probably 99.9% compatible - so runtime toggling probably isn't really needed. But if it's it is, might as well allow it here!

Reply:
Agreed, if it works at runtime without further changes, just allow it, otherwise this is fine.